.github/workflows/build.yml

```diff
@@ -66,6 +66,7 @@ jobs:
       matrix:
         worker:
           - docker
+          - docker-containerd  # same as docker, but with containerd snapshotter
```

Comment:
Could we maybe call this worker something different? Looks like it clashes with `docker-container`.

If we could use `+` in the worker name, we could do a variant style like `docker+containerd`?